### PR TITLE
Add details about logging support

### PR DESCRIPTION
**Pull request by jonahgraham**
Part of #27

### CODE REVIEW

**Comment by jonahgraham** [comment]
@HannesWell can you have a look at this and see if it meets your expectations?

**Comment by jonahgraham** [comment]
Rendered version is here: https://github.com/eclipse-packaging/packages/blob/changelog-entry-for-logging/CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ The [Eclipse IDE New & Noteworthy](https://eclipseide.org/release/noteworthy/) p
 
 ## 2023-09
 
+- Packages now contain configuration and start levels for logging support.
+This change applies the [recommendation](https://eclipse.dev/eclipse/news/4.28/platform.php#slf4j.api-version-2) from Eclipse Platform to support SLF4J version 2.
+For implementation details refer [EPP Issue #27](https://github.com/eclipse-packaging/packages/issues/27).
```
Comment (HannesWell) [inline CHANGELOG.md:12]:
```suggestion
For implementation details refer to [EPP Issue #27](https://github.com/eclipse-packaging/packages/issues/27).
```

**Review by HannesWell — APPROVED** [review]
Looks good. Thanks you.

Just one typo.

**Comment by jonahgraham** [comment]
Thanks @HannesWell for the review - this will appear on https://eclipseide.org/release/noteworthy/ when release is made next week.

**Comment by HannesWell** [comment]
> Thanks @HannesWell for the review - this will appear on https://eclipseide.org/release/noteworthy/ when release is made next week.

That's nice. Looking forward to it!

**Pull request merged**